src/renderer/chrome-api.js

```diff
@@ -99,7 +134,21 @@ exports.injectTo = function (extensionId, isBackgroundPage, context) {
     chrome.tabs.onRemoved.emit(tabId)
   });
 
+  ipcRenderer.on('cx-event', (_, e) => {
+    const { channel, payload } = e;
+    const cxEventEmitter = channel.split('.')
+      .reduce(
+        (emitter, path) => emitter = emitter[path],
+        chrome
+      )
+    const emittablePayload = isIterable(payload) ? payload : [payload];
+
+    cxEventEmitter.emit(...emittablePayload)
+  });
+
   chrome.runtime.onInstalled.emit({ reason: 'install' });
 
+  chrome = middlewareLogger(chrome);
```

Comment:
🤔not sure it's clear what is happening here and rather name the default export `middlewareLogger` in the implementation to not make it confusing.

Reply:
Yes, I will rename the default export to match the usage ;)

Reply:
Ok now I guessed it. I'd prefer we keep this for development only.

Reply:
yes, I will use `remote.app.isPackaged()` if okay for you ?

Reply:
Arf, can't it be passed as initialization variable somewhere? I'd prefer you use a env variable directly because `remote.app.isPackaged()` will make a sync back and forth to main